### PR TITLE
[MB2] Implement Wowzer, the Aspirational

### PR DESCRIPTION
**Pull request by Cguy7777**
This currently has one hint for all of the requirements to win, which seemed like cleanest approach to me, but perhaps multiple hints would be better? There are currently test failures due to not having individual hints for the monarch, city's blessing, and initiative; this is the first card to check for more than one of those conditions.

### CODE REVIEW

File: Mage.Sets/src/mage/cards/w/WowzerTheAspirational.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            new PermanentsOnTheBattlefieldCondition(new FilterControlledPermanent(SubType.BLOOD)),
+            new PermanentsOnTheBattlefieldCondition(new FilterControlledPermanent(SubType.CLUE)),
+            new PermanentsOnTheBattlefieldCondition(new FilterControlledPermanent(SubType.FOOD)),
+            new PermanentsOnTheBattlefieldCondition(new FilterControlledPermanent(SubType.MAP)),
+            new PermanentsOnTheBattlefieldCondition(new FilterControlledPermanent(SubType.POWERSTONE)),
+            new PermanentsOnTheBattlefieldCondition(new FilterControlledPermanent(SubType.TREASURE)),
+            MonarchIsSourceControllerCondition.instance,
+            CitysBlessingCondition.instance,
+            HaveInitiativeCondition.instance);
+    private static final Hint hint = new ConditionHint(
```
Comment (xenohedron) [inline Mage.Sets/src/mage/cards/w/WowzerTheAspirational.java:46]: the problem with the single hint is that it doesn't tell you what you're missing

but I don't have a strong opinion here really, if you like the way you have it, just add to SKIP_LIST_MISSING_ABILITIES in the verify test

File: Mage.Sets/src/mage/cards/w/WowzerTheAspirational.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        this.subtype.add(SubType.WURM);
+        this.power = new MageInt(10);
+        this.toughness = new MageInt(10);
+
+        // Whenever Wowzer, the Aspirational attacks,
+        // if you have an {E}, control a Blood, a Clue, a Food, a Map, a Powerstone, and a Treasure,
+        // are the monarch, and have the city's blessing and the initiative, you win the game.
+        this.addAbility(new ConditionalInterveningIfTriggeredAbility(
+                new AttacksTriggeredAbility(new WinGameSourceControllerEffect()),
+                new CompoundCondition(condition),
```
Comment (xenohedron) [inline Mage.Sets/src/mage/cards/w/WowzerTheAspirational.java:65]: why wrap it in another compound condition?

Comment (Cguy7777) [reply Mage.Sets/src/mage/cards/w/WowzerTheAspirational.java]: Good catch, I think I moved some code around and forgot to remove that.

File: Mage.Sets/src/mage/cards/w/WowzerTheAspirational.java (hunk 1 above)
Comment (Cguy7777) [reply Mage.Sets/src/mage/cards/w/WowzerTheAspirational.java]: After giving it a try, I agree that having individual hints is much better. Thank you. :-)

**Review by xenohedron — APPROVED** [review]

**Pull request merged**